packages/server/src/datasources/dataset-service.ts

```diff
@@ -70,7 +70,7 @@ export class DatasetService extends DataSource {
   }
 
   async getDatasetImages(id: string): Promise<LabeledImage[]> {
-    // return this.listImages(id);
+    // return this.listImages(id); //<-- supressed due to "too many open files error"
     return [];
```

Comment:
Is returning an empty list correct final, or a TODO based on error mentioned at line 73

Reply:
definitely a todo, we want to return some images, just not open all of them at once